### PR TITLE
Editors Update - Adding additional troubleshooting steps for Visual Studio Code

### PR DESCRIPTION
**Pull request by getHashSet**
Additional Troubleshooting steps added for Visual Studio Code users that are unable to grant $PATH access to their VSCode instances.

Additional steps to resolve include opening a new instance of VSCode through the built in terminal windows.

Added code examples and testing notes.

### CODE REVIEW

File: book/docs/setup/editors/vscode.md [hunk 1]
```diff
@@ -17,4 +18,60 @@ _This guide assumes that you've completed the steps described in [Prerequisites]
...
+foo x y z =  readMaybe x `bindMaybe` \num1 ->
+             readMaybe y `bindMaybe` \num2 ->
+             readMaybe z `bindMaybe` \num3 ->
+             Just (num1 + num2 + num3)
+```
+
+### Possible Solution
+1. Open the built-in terminal inside your Visual Studio Code by pressing `control` _+_ `~ ` at the same time or by going to `Terminal > New Terminal` within the file menu.
+
+2. From the built in terminal, navigate to your **plutus** directory or clone a new copy of the [plutus repository](https://github.com/input-output-hk/plutus).
```
Comment (rikgirbes) [inline book/docs/setup/editors/vscode.md]:
```suggestion
2. From the built-in terminal, navigate to your **plutus** directory or clone a new copy of the [plutus repository](https://github.com/input-output-hk/plutus).
```

Keeping consistent with 1.

File: book/docs/setup/editors/vscode.md [hunk 2]
```diff
@@ -17,4 +18,60 @@ _This guide assumes that you've completed the steps described in [Prerequisites]
...
+Ubuntu                                          v20.x
+Windows                                         v10.x
+Nix                                             v2.3.10
+Visual Studio Code                              v1.55.2
+VSCode Extension Haskell                        v1.2.0
+VSCode Extension Haskell Syntax Highlighting    v3.4.0
+```
+
+### How to Check
+_Select a function in the Visual Studio Code Editor and right click and choose **Go to Definition** or highlight a function and press **F12**_.
```
Comment (rikgirbes) [inline book/docs/setup/editors/vscode.md]:
```suggestion
_Select a function in the Visual Studio Code Editor and right-click and choose **Go to Definition** or highlight a function and press **F12**_.
```

File: book/docs/setup/editors/vscode.md [hunk 3]
```diff
@@ -17,4 +18,60 @@ _This guide assumes that you've completed the steps described in [Prerequisites]
    ```
    _Note: if you're using `nix-shell`, make sure to run it first._
 
-If everything is fine you should get auto-completion and other features working 🎉
\ No newline at end of file
+If everything is fine you should get auto-completion and other features working 🎉
+
+---
+</br>
```
Comment (rikgirbes) [inline book/docs/setup/editors/vscode.md]: </br> redundant

**Review by rikgirbes — CHANGES_REQUESTED** [review]
VERY little feedback 😊 

Otherwise: Awesome, thanks! 👍🏽

File: book/docs/setup/editors/vscode.md (hunk 1 above)
Comment (getHashSet) [reply book/docs/setup/editors/vscode.md]: I totally agree! Resolved in commit c1a467a on `getHashSet/docs.plutus-community.com` branch `editors`.

File: book/docs/setup/editors/vscode.md (hunk 2 above)
Comment (getHashSet) [reply book/docs/setup/editors/vscode.md]: Good catch. Resolved in commit c1a467a on `getHashSet/docs.plutus-community.com` branch `editors`.

File: book/docs/setup/editors/vscode.md (hunk 3 above)
Comment (getHashSet) [reply book/docs/setup/editors/vscode.md]: Bad habits die hard. I have removed the `</br>` line break. Resolved in commit c1a467a on `getHashSet/docs.plutus-community.com` branch `editors`.

**Review by rikgirbes — APPROVED** [review]

**Pull request merged**